presto-spi/src/main/java/com/facebook/presto/spi/security/AccessDeniedException.java

```diff
@@ -228,14 +228,14 @@ public static void denyRevokeTablePrivilege(String privilege, String tableName,
         throw new AccessDeniedException(format("Cannot revoke privilege %s on table %s%s", privilege, tableName, formatExtraInfo(extraInfo)));
     }
 
-    public static void denyShowGrants(String user)
+    public static void denyShowGrants()
     {
-        denyShowGrants(user, null);
+        denyShowGrants(null);
     }
 
-    public static void denyShowGrants(String user, String extraInfo)
+    public static void denyShowGrants(String extraInfo)
```

Comment:
How do you know that "on" would be the proper formatting for this?
Saying the user still seems helpful.